imports/server/profile.ts

```diff
@@ -22,23 +21,16 @@ Meteor.methods({
       muteApplause: Boolean,
       dingwords: [String],
     });
-    const user = MeteorUsers.findOne(this.userId)!;
-    const primaryEmail = user.emails && user.emails[0].address;
-
     Ansible.log('Updating profile for user', { user: this.userId });
-    Profiles.update({
+    Meteor.users.update({
```

Comment:
Got some more uses of `Meteor.users` in this file -- is this just inconsistency, or is there a set of situations where we need to use `Meteor.users` directly instead of `MeteorUsers`?